operators/advanced_network/README.md

```
@@ -24,15 +24,16 @@ is available in userspace, thus bypassing the kernel's networking stack entirely
 #### Requirements
 
 - Linux
-- An NVIDIA NIC with a ConnectX-6 or later chip
+- An NVIDIA NIC with a ConnectX6-Dx or later chip
 - System tuning as described [here](/tutorials/high_performance_networking/README.md)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)